src/index.js

```diff
@@ -54,5 +73,13 @@ taskContainer.addEventListener('click', (e) => {
    }
  });
 
+ clearAllDone.addEventListener('click',(e)=>{
+  const data= Operations.getAllTasks();
+  const storage=data.filter((todo)=>todo.completed===false);
+  console.log(storage)
```

Comment:
- [ ] Kindly remove this unnecessary `console.log()`. It does not make your codebase look professional. It is used for debugging.

Reply:
done